front/app/api/notifications/types.ts

```diff
@@ -188,16 +188,19 @@ export interface IMentionInCommentNotificationData
   };
 }
 
-type InternalCommentType =
-  | 'mention_in_internal_comment'
-  | 'internal_comment_on_your_internal_comment'
-  | 'internal_comment_on_idea_assigned_to_you'
-  | 'internal_comment_on_initiative_assigned_to_you'
-  | 'internal_comment_on_idea_you_moderate'
-  | 'internal_comment_on_idea_you_commented_internally_on'
-  | 'internal_comment_on_initiative_you_commented_internally_on'
-  | 'internal_comment_on_unassigned_unmoderated_idea'
-  | 'internal_comment_on_unassigned_initiative';
+export const internalCommentTypes = [
```

Comment:
```suggestion
export const internalCommentNotificationTypes = [
```

Reply:
Would this const fit better in campaigns/types.ts? Since we use it to pass to a paramater 'withoutCampaigns' and a hook called useCampaignConsents. That could reduce confusion too.